source_code/main_mcu/src/TIMER/driver_timer.c

```diff
@@ -221,6 +221,54 @@ void timer_set_calendar(uint16_t year, uint16_t month, uint16_t day, uint16_t ho
 #endif
 }
 
+/*! \fn	time(void)
+*   \brief  Get the current unix time
+*
+*   Compute the number of seconds since EPOCH (Jan 1st. 1970), also
+*   commonly known as UNIX time. UNIX time does NOT include LEAP seconds.
+*   See https://pubs.opengroup.org/onlinepubs/9699919799/xrat/V4_xbd_chap04.html#tag_21_04_16
+*   \param  None
+*   \return Seconds since Jan 1st. 1970
+*/
+uint64_t time(void)
```

Comment:
can you rename it into driver_timer_get_time , as time() is already defined when compiling for the emulator?

Reply:
Done!